### ISSUE TITLE
Tab orders not enforced

### ISSUE THREAD

**Issue by samuelfmlourenco** [bloguetronica/cp2130-conf#22]
No windows have tab orders being enforced in order to follow a specific, coherent sequence. This means that, for any window, the tab order is determined only by the order in which the widgets (buttons, boxes, etc) were added during the window design process. Currently, many of the application windows do not offer a sequential tab order, and this should be corrected.

**Comment by samuelfmlourenco** [comment]
Implemented in version 1.4.
